.github/workflows/Repology.yml

```diff
@@ -31,7 +31,7 @@ jobs:
         id: cpr
         uses: peter-evans/create-pull-request@v7
         env:
-          GH_TOKEN: ${{ secrets.CREW_PR_TOKEN }}
+          GH_TOKEN: ${{ secrets.CREW_PR_OPS_TOKEN }}
```

Comment:
Changing the token didn't work in my previous attempt. Specifying the token in the `peter-evans/create-pull-request@v7` action below fixes it (see `fix-repology` branch)

Reply:
I have opened a new PR in #11786 for it...